rados/conn.go

```diff
@@ -280,6 +281,38 @@ func (c *Conn) DeletePool(name string) error {
 	}
 }
 
+// TODO
+func (c *Conn) GetPoolByName(name string) (int64, error) {
+	if err := c.ensure_connected(); err != nil {
+		fmt.Println("NOT CONNECTED WHOOPS")
```

Comment:
need to remove  Println

Reply:
That is just copy/paste from other functions...